bin/load_env_secrets.sh

```diff
@@ -81,7 +81,13 @@ if [[ -n $VAULT_SECRET_PATH ]] ; then
   do
     echo "setting value for: $key"
     curr_val=$(echo $VALS | jq .data | jq --raw-output .$key)
-    export $key=$curr_val
+    # honor PORTAL_NAMESPACE from the environment, if present
```

Comment:
are we quite sure there aren't any lingering PORTAL_NAMESPACE variables in the prod environment?  I assume you've thought of this, but wanted to double-check :)

Reply:
`$PORTAL_NAMESPACE` is the only variable that is defined both in `vault` and can be passed into `load_env_secrets.sh`.  Everything else is sourced from `vault`, or passed only as an argument (like `$PASSENGER_APP_ENV`).